routes/auth.js

```diff
@@ -213,6 +213,18 @@ router.post(
   })
 );
 
+router.post(
+  '/demo',
+  csrfProtection,
+  validateLogin,
+  asyncHandler(async (req, res, next) => {
+    const user = await User.findOne({ where: { firstName: 'demo' } });
```

Comment:
perhaps setting the where constraint to a column with a unique constraint (i.e email) would be beneficial to ensure we dont mistakenly grab any other user that might be named 'demo'

Reply:
Absolutely! That makes a lot of sense.

Reply:
Yeah I agree with Austin's feedback